### PR TITLE
Add NFS Server Provisioner service

### PR DESCRIPTION
**Pull request by alembiewski**
Resolves [D2IQ-81559](https://jira.d2iq.com/browse/D2IQ-81559).

This PR adds the `nfs-server-provisioner` based on https://github.com/mesosphere/charts/tree/master/staging/nfs-server-provisioner chart to Kommander applications.

### CODE REVIEW

File: services/nfs-server-provisioner/0.6.0/defaults/cm.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+---
```
Comment (mikolajb) [inline services/nfs-server-provisioner/0.6.0/defaults/cm.yaml:2]:
```suggestion
---
```

File: services/nfs-server-provisioner/metadata.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+displayName: NFS Server Provisioner
+description: NFS Server Provisioner is an out-of-tree dynamic provisioner for Kubernetes. You can use it to quickly & easily deploy shared storage that works almost anywhere.
+type: platform
```
Comment (mikolajb) [inline services/nfs-server-provisioner/metadata.yaml:3]:
```suggestion
type: catalog
```

**Review by mikolajb — APPROVED** [review]
Great, thank you Alex!

**Comment by gracedo** [comment]
@mesosphere-mergebot backport release-2.1

**Pull request merged**